#ifndef CUDA_DIAGNOSTIC_H_INCLUDED
#define CUDA_DIAGNOSTIC_H_INCLUDED

//#include <diagnostic_updater/diagnostic_updater.h>
//#include <std_msgs/Bool.h>
//#include <diagnostic_updater/publisher.h>

class cuda_diagnostic
{
    public:
        cuda_diagnostic();
        ~cuda_diagnostic();
};
#endif  //CUDA_DIAGNOSTIC_H_INCLUDED